File: Round1/A/A.r
num <- scan("stdin", what = integer(), quiet = TRUE)       
cas <- num[1]
ans <- integer(cas)
for (i in 1:cas) {
    if (num[i + 1] == 1) {
        ans[i] = 1
    } else {
        ans[i] = num[i + 1] %/% 2
    }
}
write(as.integer(ans), file = "", sep = "\n")

File: Round1/A/A.js
var fs = require('fs');
var buf = '';

process.stdin.on('readable', function() {
    var chunk = process.stdin.read();
    if (chunk) buf += chunk.toString();
});

process.stdin.on('end', function() {
    let data = buf.split('\n');
    let cas = parseInt(data[0]);
    for (var i = 1; i <= cas; ++i) {
        let num = parseInt(data[i]);
        console.log(num == 1 ? 1 : num / 2 | 0);
    }
});
